src/docs/device_readme.md

```diff
@@ -1,4 +1,20 @@
-# Usage
-Make sure iotedge is installed on this device.  [[TODO: Link iotedge install docs here]]
+# Prerequisites
+Each device must have IoT Edge (must be v1.2 or later) installed. Pick the [supported OS](https://docs.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11) and follow the [tutorial](https://docs.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11) to install Azure IoT Edge.
 
-From this folder, run `./install.sh`. This will copy the certificated, configure iot edge, and restart iotedge.
+Alternatively, if the OS is Tier1 supported OS and using DPS SAS Key provisioning, run [this installation script](https://github.com/Azure/iot-edge-config) can reduce the steps to install and provision.
+
+# Steps
+
+1. After install and configure IoT Edge to Azure IoT Hub or Azure IoT Central, copy the zip file for each device created, named [[device-id]].zip. 
+2. Transfer each zip to its respective device. A good option for this is to use [scp](https://man7.org/linux/man-pages/man1/scp.1.html).
+3. Unzip the zip file by running following commands
+
+```Unzip
+    sudo apt install zip
+    unzip ~/<PATH_TO_CONFIGURATION_BUNDLE>/<CONFIGURATION_BUNDLE>.zip
+```
+4. Run the script
+```Run
+    sudo ./install.sh
+```
+5. Follow the prompt by entering the hostname (FQDN or IP address). On the parent device, it will prompt the hostname and on the child deivce, it will prompt both the hostname of the child and parent device.

```

Comment:
```suggestion
5. If the hostname was not provided in the configuration file, it will prompt for hostname. Follow the prompt by entering the hostname (FQDN or IP address). On the parent device, it may prompt its own hostname and on the child deivce, it may prompt the hostname of both the child and parent device.

```